databaseinit/user.sql

```diff
@@ -0,0 +1,3 @@
+CREATE USER 'libraryuser'@'localhost' IDENTIFIED BY 'librarypassword';
```

Comment:
This will fail if user already exists - can you drop the user first - https://dev.mysql.com/doc/refman/5.0/en/drop-user.html